### PR TITLE
Economy tests refactor

### PR DESCRIPTION
**Pull request by JonaszJestem**
Migrated tests to junit4. Replaced use of deprecated methods with correct ones. Increased test coverage. Overall refactor.

### CODE REVIEW

File: Essentials/test/com/earth2me/essentials/EconomyTest.java [hunk 1]
```diff
@@ -105,7 +105,6 @@ public void testNPC() throws UserDoesNotExistException {
         assertTrue("NPC creation should be possible", Economy.createNPC(NPC));
         assertTrue("NPC exists after creation", Economy.playerExists(NPC));
         assertNotNull("NPC can be accessed after creation", ess.getOfflineUser(NPC));
-        assertTrue(Economy.isNPC(NPC));
```
Comment (mdcfe) [inline Essentials/test/com/earth2me/essentials/EconomyTest.java]: Why was this line removed?

**Review by mdcfe — CHANGES_REQUESTED** [review]

**Comment by Ichbinjoe** [comment]
There hasn't been activity on this PR for quite some time - closing.
